pixart.js

```diff
@@ -1 +1,21 @@
+var brushBox = document.querySelector(".brush");
+var inputBox = document.innerHTML.querySelector("#color-field").value;
```

Comment:
This line triggers an error which stops your program. 

Important: you should be constantly refreshing your page and checking for errors. If you get an error on line two (such as this) you need to *fix this error* before proceeding.